src/components/UserActions/UserActivity.ts

```diff
@@ -289,37 +292,56 @@ export class UserActivity extends Component {
         li.classList.add(CUSTOM_EVENT_CLASS);
 
         const titleElem = document.createElement('div');
-        titleElem.classList.add(TITLE_CLASS);
+        titleElem.classList.add(ACTIVITY_TITLE_CLASS);
         titleElem.innerText = `${l(action.raw.event_type || `${UserActivity.ID}_custom`)}`;
 
+        const titleSection = this.buildTitleSection(action);
+        titleSection.querySelector(`.${ACTIVITY_TITLE_CLASS}`).remove();
+        titleSection.prepend(titleElem);
+
         const dataElement = document.createElement('div');
         dataElement.classList.add(DATA_CLASS);
         dataElement.innerText = action.raw.event_value || '';
 
-        li.appendChild(titleElem);
+        li.appendChild(titleSection);
         li.appendChild(dataElement);
-        li.appendChild(this.buildTimestampElement(action));
+        li.appendChild(this.buildOriginElement(action));
         li.appendChild(this.buildIcon(dot));
 
         return li;
     }
 
-    private buildTimestampElement(action: UserAction) {
+    private buildOriginElement(action: UserAction): HTMLElement {
         const el = document.createElement('div');
-        el.classList.add(TIMESTAMP_CLASS);
-        el.innerText = `${formatTime(action.timestamp)}${(action.raw.origin_level_1 && ` - ${action.raw.origin_level_1}`) || ''}`;
+        el.classList.add(ORIGIN_CLASS);
+        el.innerText = `${(action.raw.origin_level_1 && action.raw.origin_level_1) || ''}`;
```

Comment:
Huh? I don't understand this line.

Reply:
me neither that's just the way it was :P I'll try removing it and we'll see if the sky falls.

Reply:
The tests pass just fine without it.

Reply:
@jeremierobert-coveo you worked first on this component iirc, can you check wassup?

Reply:
This was rendering the time. Can you check that the now string is something like `<My Datetime> - <My origin lvl 1>`

Reply:
we moved the timestamp to be on the right, inline with the title. So now it just renders the origin level. What we're curious about is this: 
`action.raw.origin_level_1 && action.raw.origin_level_1 || ''`

Reply:
👌